### PR TITLE
[API,CLI](CIVO) Merging ha_civo/ and civo/

### PR DESCRIPTION
**Pull request by dipankardas011**
# Tasks / Issue :construction:
Merge the api/civo, api/ha_civo -->>> api/civo

# Solution :heavy_check_mark:
after the merge a lot of redundancy is eliminated

# Screenshots :framed_picture:

# Note to reviewers :notebook:

### CODE REVIEW

**Comment by AvineshTripathi** [comment]
can we have a UI for the coverage report?

**Comment by dipankardas011** [comment]
> can we have a UI for the coverage report?

codecov but we have to integrate it with this repo

we can format the above txt to make it look good

**Comment by dipankardas011** [comment]
## Note
Made credential files to json file rather than using simple text files

File: cli/cmd/credentials.go [hunk 1]
```diff
@@ -48,8 +48,9 @@ var credCmd = &cobra.Command{
 			return
 		}
 
-		for !isSuccess {
-			isSuccess = storeCredentials(choice)
+		isSuccess = storeCredentials(choice)
+		if !isSuccess {
+			fmt.Println("Login Failed")
```
Comment (dipankardas011) [inline cli/cmd/credentials.go:53]: Add return or break statement

**Review by dipankardas011 — COMMENTED** [review]
Remove the comments

**Pull request merged**